cves/CVE-2015-3183.yml

```diff
@@ -155,12 +156,12 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: I didn't see any significant events based on the relative research I found.
```

Comment:
Perhaps "relative" should become "related", I am not the best at grammar, but i think this makes a bit more sense.